index.js

```diff
@@ -136,27 +136,18 @@ module.exports = function(source) {
                         });
                     });
 
-                    let defaultTech;
-
                     // Each tech has own generator
                     const res = Object.keys(extToFiles)
                         // use techs from config for order
                         // so the first one would be default, `js` in most cases
                         .sort((a, b) => techs.indexOf(extToTech[a]) - techs.indexOf(extToTech[b]))
-                        .map((ext, i) => {
+                        .map((ext) => {
                             const tech = extToTech[ext] || ext;
-                            i || (defaultTech = tech);
-                            return `exportObj['${tech}'] = ${(generators[tech] || generators['*'])(extToFiles[ext])};`;
+                            return `${(generators[tech] || generators['*'])(extToFiles[ext])};`;
                         })
                         .join('\n');
 
-                    node.update([
-                        '(function(exportObj) {',
-                        res,
-                        `exportObj.default = exportObj['${defaultTech}'];`,
-                        'return exportObj;',
-                        '}({ __esModule: true }))'
-                    ].join('\n'));
+                    node.update(`(function() {var defTech = ${res} return defTech;})()`);
```

Comment:
why not just
```js
node.update(`(function() { return ${res}; })()`);
```
?

or even more — `node.update(res);`?

Reply:
It should be expended into 
```js
node.update(`(function() {var defTech = %firstTechGeneratorOuput%; %secondTechGeneratorOuput%; %thirdTechGeneratorOuput%; return defTech;})()`);
```
With the solution i am going to get JS as first resolved tech (by default), but return that one.

Reply:
👍

Reply:
@godfreyd Can you try this PR? Does it work for you?

Reply:
@Yeti-or Does it look fine for you?

Reply:
I'll check it tonight

Reply:
Since we exporting only `firstTechGeneratorOuput` as `defTech` I don't see any sense in `iife`
we should use `[ %firstTechGeneratorOuput%, %secondTechGeneratorOuput%, %thirdTechGeneratorOuput%][0]`

Reply:
@nikita-ezhkov could you change it?

Reply:
@Yeti-or Sure. I will mention you when it is done